stateMachineClass.py

```diff
@@ -4,14 +4,31 @@
 #There are test functions at the end. 
 #Note that to externally activate a CA State, you should only send
 #excitatory connections to the first 8 neurons.
+#Using the nealNRPCover so no nealParams
```

Comment:
If this does not work against the tests we should revert this back or make fix it completly (but this will be quite a big job).